hearts/game/hearts.py

```diff
@@ -458,13 +459,21 @@ def serialize(self, for_player=None):
             player = for_player if isinstance(for_player, Player) else Player(for_player)
             hands = {player.username: self.hands[player].serialize()}
 
+        def serialize_pass_selections(pass_selections):
```

Comment:
This should also have a `for_player` option, so that we don't expose who passed what to everyone.